content/blog/configuring-azure-ad-with-greenlake-cloud-platform-and-aruba-central.md

```diff
@@ -0,0 +1,236 @@
+
+I'll explain the process for configuring Azure AD to authenticate users into Greenlake Cloud Platform (GLCP) and Aruba Central using SAML IDP.
+
+If you're looking for the Okta version of this guide can be found on [WIFI-GUYS](https://www.wifi-guys.com/?p=512)
+<!-- prettier-ignore-start -->
+## Contents
+
+- [Configuring Azure AD as the SAML IDP with Greenlake Cloud Platform and Aruba Central](#configuring-azure-ad-as-the-saml-idp-with-greenlake-cloud-platform-and-aruba-central)
+	- [Contents](#contents)
+	- [Before you Begin](#before-you-begin)
+	- [Terms used in this blog post](#terms-used-in-this-document)
+	- [Steps to Configure SSO/SAML Application in Azure AD](#steps-to-configure-ssosaml-application-in-azure-ad)
+	- [Step 1: Create an Azure AD Enterprise Application](#step-1-create-an-azure-ad-enterprise-application)
+	- [Step 2: Configure GCLP for SAML Federation](#step-2-configure-gclp-for-saml-federation)
+	- [Login to GLCP and Aruba Central using Azure AD](#login-to-glcp-and-aruba-central-using-azure-ad)
+	- [Using Azure AD MFA](#using-azure-ad-mfa)
+	- [Troubleshooting](#troubleshooting)
+	- [Appendix: Generating the hpe_ccs_attribute](#appendix-generating-the-hpe_ccs_attribute)
+<!-- prettier-ignore-end -->
+
+## Before you Begin
+This blog references the following documentation:
+
+* [HPE Greeenlake User Guide](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=ccs-help_en_us)
+
+* [Single sign-on (SS) authentication](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=a00092451en_us&page=GUID-CD81FAF8-9601-4773-899F-049A506FEE2E.html)
+
+* [HPE Greenlake Platform Guide](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=a00120892en_us)
+
+If you're looking for the Central 2.5.4 SAML integration guide, [it has been moved](https://github.com/michaelrosejr/arubasso/tree/main/Central254).
+
+## Terms used in blog post
+
+- `CCS:` Common Cloud Service
+- `GLPC:` GreenLake Cloud Platform
+- `SSO:` Single Sign On
+- `SAML:` Security Assertion Markup Language
+- `AD:` Active Directory 
+- `MFA:` Multi-Factor Authentication
+- `MSP:` Managed Service Proivder
+- `XML:` eXtensible Markup Language
+
+
+## Steps to Configure SSO/SAML Application in Azure AD
+To configure SSO in Aruba Central, first download the metadata file from Azure AD.
+
+
+<ol>
+	<li>Create an Enteprise Application in the [Azure Portal](https://portal.azure.com)</li>
+	<li>Configure the Enterprise Application for GLCP</li>
+	<li>Download the federated metadata XML file from Enterprise Application</li>
+	<li>Claim and Configure your domain within GLCP</li>
+	<li>Upload the federated metadata XML file to GLCP </li>
+	<li>Create recovery account</li></ol>
+
+
+
+## Step 1: Create an Azure AD Enterprise Application
+
+
+* Log into to Azure portal.
+
+* Click **Enterprise Applications** (you may need to search for it, if it's not on your menu)
```

Comment:
Seeing as how you've written this here, my guess is that this is what the user sees on the screen, so having initial capital letters is okay if that's how it's presented.